cypress/integration/article_spec.js

```diff
@@ -1,10 +1,30 @@
-import { testResponseCode } from './test-helper';
+import {
+  testResponseCode,
+  getElement,
+  shouldContainText,
+  shouldContainStyles,
+} from './test-helper';
 
 describe('News Article', () => {
   beforeEach(() => {
     cy.visit('/');
   });
 
+  it('should render the BBC News branding', () => {
+    const headerElement = getElement('header');
+    shouldContainStyles(headerElement, 'height', '40px');
+    shouldContainStyles(headerElement, 'background-color', 'rgb(187, 25, 25)');
```

Comment:
Is it possible to using hexadecimal values instead of rgb? Mainly for consistency as we set the colour in hex.

Reply:
As discussed Cypress only supports RGB and doesn't parse HEX so this can't be changed, happy as is